laghos.cpp

```diff
@@ -757,17 +757,47 @@ int main(int argc, char *argv[])
         // -restore phase
         // No need to specify t_final because the loop in -restore phase is determined by the files in ROMsol folder.
         // When -romhr or --romhr are used in -online phase, then -restore phase needs to be called to project rom solution back to FOM size
+        std::ifstream infile_tw_steps("run/tw_steps");
+        int nb_step;
+        nb_step = 1000000;
```

Comment:
Why does this need to be initialized to a large number? Is it checked later whether this is overwritten to a smaller number? Also, the initialization and declaration can be on one line.

Reply:
Thank you for pointing it out, Dylan. Initializing nb_step to zero is completely fine because ti starts with 1. I have changed it.